### PR TITLE
Correct path and correctly escape path when running windows install.

### LINKED ISSUE #3809: fine install command fails on Windows 10
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same problem, before opening a new one.
- [X] This issue only relates to a single bug. I will open new issues for any other problems.

### Describe the bug

On Windows 10 "fyne install" makes dir in Program File and shortcut in Start Menu but don't move built exe from src folder to Program Files

### How to reproduce

1. Run "fyne install” in source dir 
2. It makes folder in Program files and  shortcut in Start Menu
3. Executable remain in source dir but not been moved in correspondent folder of  Program files

### Screenshots

_No response_

### Example code

```go
package main

import (
	"fyne.io/fyne/v2/app"
	"fyne.io/fyne/v2/container"
	"fyne.io/fyne/v2/widget"
)

func main() {
	a := app.New()
	w := a.NewWindow("Hello")

	hello := widget.NewLabel("Hello Fyne!")
	w.SetContent(container.NewVBox(
		hello,
		widget.NewButton("Hi!", func() {
			hello.SetText("Welcome :)")
		}),
	))

	w.ShowAndRun()
}
```

### Fyne version

fyne version v2.3.1

### Go compiler version

go1.20.2 windows/amd64

### Operating system and version

Windows 10

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by Bluebugs**
### Description:
Make sure we actually copy the newly created executable from the right location to where we want it to be installed.

Fixes #3809

### Checklist:
- [ ] Lint and formatter run with no errors.
- [ ] Tests all pass.

### CODE REVIEW

File: cmd/fyne/internal/commands/package-windows.go [hunk 1]
```diff
@@ -131,8 +135,15 @@ func (p *Packager) packageWindows(tags []string) error {
 func runAsAdminWindows(args ...string) error {
 	cmd := "\"/c\""
 
-	for _, arg := range args {
-		cmd += ",\"" + arg + "\""
+	for idx, arg := range args {
+		if idx == 0 {
```
Comment (andydotxyz) [inline cmd/fyne/internal/commands/package-windows.go:139]: This can't be right? Only adding a comma for 0 index?
I think this would benefit from a unit test showing the correct format.

**Review by andydotxyz — CHANGES_REQUESTED** [review]
It doesn't seem like the right encoding but it's hard to tell without a little test confirming how it comes together.

File: cmd/fyne/internal/commands/package-windows.go (hunk 1 above)
Comment (Bluebugs) [reply cmd/fyne/internal/commands/package-windows.go:139]: Actually cmd.exe does parse the string given to it as the command to execute. So it means the entire arguments list has to be in the same argument passed to it. It is closed at the end of the loop, if there was more than one. I will see to add a unit test.

Comment (andydotxyz) [reply cmd/fyne/internal/commands/package-windows.go:139]: As requested above can you please add a unit test?
The mix of single and double quotes is hard to parse here. With args "1" and "2" it looks like it will produce: 
* (before) `"/c","1","2"`
* (after) `"/c" ,'1 "2"'`

Without seeing the expected output I can't figure if this is desired...

**Review by andydotxyz — CHANGES_REQUESTED** [review]
As in other message, please add a unit test to show the expected output to confirm the formatting.

**Review by andydotxyz — APPROVED** [review]
Thanks. The examples demonstrate why the seemingly peculiar mix if brackets and raw parameters works.

I do have a concern that without escaping the first parameter it may fall down on some things (like running a command/binary that is not builtin) but it probably doesn't apply here.
